### PR TITLE
make some variables non-optional in Python.py

### PR DESCRIPTION
**Pull request by alexboche**
These commands were causing some misfires so I made the variables required rather than optional

### CODE REVIEW

**Comment by kendonB** [comment]
We should get agreement from someone who uses caster for python programming. @mrob95?

**Comment by alexboche** [comment]
> We should get agreement from someone who uses caster for python programming. @mrob95?

The commands I'm changing are commands that I added pretty recently, so there's a good chance that no one even uses them yet.

**Comment by LexiconCode** [comment]
Since these are commands you recently added go ahead and make the changes.

File: castervoice/lib/ccr/python/python.py [hunk 1]
```diff
@@ -120,19 +120,19 @@ class Python(MergeRule):
             # going into the command pallet (cs-p) and typing in "insert snippet"
             # then press enter and then you have choices of snippets show up in the drop-down list.
             # you can also make your own snippets.
-        "try [<exception>]":
+        "try <exception>":
             R(Text("try : ") + Pause("10") + Key("enter/2")
```
Comment (kendonB) [inline castervoice/lib/ccr/python/python.py]: no space here after try. There should be two enters (why a 0.02 second pause??). The cursor should end up on the line below try

Comment (kendonB) [reply castervoice/lib/ccr/python/python.py]: jason above also needs fixing. Can you do that here?

File: castervoice/lib/ccr/python/python.py [hunk 2]
```diff
@@ -120,19 +120,19 @@ class Python(MergeRule):
...
-        "try [<exception>] as":
+        "try <exception> as":
             R(Text("try :") + Pause("10") + Key("enter/2") + Text("except %(exception)s as :")
             + Pause("10") + Key("enter/2")),
 
         # class and class methods
         "subclass": R(Text("class ():") + Key("left:3")),
         "dunder": R(Text("____()") + Key("left:4")),
         "init": R(Text("__init__()") + Key("left")),
-        "meth [<binary_meth>]": R(Text("__%(binary_meth)s__(self, other):")),
```
Comment (kendonB) [inline castervoice/lib/ccr/python/python.py]: Add enter characters after these?

**Comment by LexiconCode** [comment]
Will be invalidated by merger of the rewrite. Reopen under a new pull request.

**Pull request closed without merging**